### DISCUSSION TITLE
Strelka processed files 2.3.50

### DISCUSSION THREAD

**Discussion by g2dbone** [Security-Onion-Solutions/securityonion#4458] [Unsupported Versions]
Are files within the /nsm/strelka/processed/ directory supposed to get deleted at some point?  Currently I have a sensor that has accumulated 2.3T of files within that directory.

**Comment by weslambert** [comment]
Yes, there is a cron job that runs every minute to delete files from that directory if the disk threshold is being reached.  Is this not the case? It could also be that these files aren't being deleted, but certain other files that are also considered within the cron job (so-sensor-clean) that are being removed before these files.
